ci/prs.yml

```diff
@@ -11,6 +11,51 @@ pr:
       - release/*
 
 jobs:
+
+- job: test_windows_signing
```

Comment:
I assume you’re still debugging this which is why it is in the PRs build?

Reply:
Yes. Currently, on main, it looks like it's not running at all. Trying to debug this on main is not only super inconvenient, it's also a bit dangerous. By running this on a PR, I can print out whatever I want as the secrets are just not available to the pipeline to start with.